src/core/renderer_gl/renderer_gl.cpp

```diff
@@ -979,6 +1032,11 @@ void RendererGL::screenshot(const std::string& name) {
 }
 
 void RendererGL::clearShaderCache() {
+	if (asyncCompiler && shaderMode == ShaderMode::Hybrid) {
```

Comment:
Nit but the conditions should be reversed and `asyncCompiler != nullptr` instead of having people guess what async compiler means